src/core/attributeformmodelbase.cpp

```diff
@@ -239,6 +240,72 @@ void AttributeFormModelBase::updateAttributeValue( QStandardItem *item )
     //set item visibility to false in case it's a linked attribute
     item->setData( !mFeatureModel->data( mFeatureModel->index( fieldIndex ), FeatureModel::LinkedAttribute ).toBool(), AttributeFormModel::CurrentlyVisible );
   }
+  else if ( item->data( AttributeFormModel::ElementType ) == QStringLiteral( "qml" ) )
+  {
+    QString qmlCode = mEditorWidgetCodes[item];
+    int openPos = qmlCode.indexOf( QStringLiteral( "expression.evaluate(" ), 0, Qt::CaseInsensitive );
```

Comment:
rather than manually looping in the text, I would rather use a regex: 
https://stackoverflow.com/a/19863847/1548052

You have to first remove the heading "expression.evaluate" and feed the regex with the remaining text:
https://regex101.com/r/eBtSTM/129

This regex (or your code) is not 100% robust as you could have unbalanced parenthesis in some strings (let's say a list of items: "a) … b)… c) …" ). But it might be acceptable.

Reply:
@3nids , I went down this road until I realized - while chatting with @m-kuhn  - that we can use an unescaped double quote as end delimiter. So yeah, regexp is the way forward. Modifying that now.

Reply:
nice catch, cool